static/swagger-specs/destination-authoring.yaml

```diff
@@ -1662,6 +1662,16 @@ definitions:
             example: true
             title: "Is Identity Required"
             description: "Use true if you users should be able to map identity namespaces from Experience Platform to your desired schema."
+          segmentNamespaceAllowList:
+            type: "array"
+            example: ["AudienceManager"]
+            title: "Segment Namespace Allow List"
+            description: "Allows users to map only audiences from the audience namespaces defined in the array to the destination. <br><br> Use of this parameter is discouraged in most cases. Instead, use `\"segmentNamespaceDenyList\":[]` to allow all types of audiences to be exported to your destination. <br><br> If both `segmentNamespaceAllowList` and `segmentNamespaceDenyList` are missing from your configuration, users will only be able to export audiences originating from the [Segmentation Service](https://experienceleague.adobe.com/en/docs/experience-platform/segmentation/home). <br><br>`segmentNamespaceAllowList` and `segmentNamespaceDenyList` are mutually exclusive."
+          segmentNamespaceDenyList:
+            type: "array"
+            example: ["AudienceManager"]
+            title: "Segment Namespace Deny List"
+            description: "Restricts users from mapping audiences from the audience namespaces defined in the array to the destination. <br><br>To allow the export of all audiences, regardless of the origin, set `\"segmentNamespaceDenyList\":[]`. <br><br>If both `segmentNamespaceAllowed` and `segmentNamespaceDenyList` are missing from your configuration, users will only be able to export audiences originating from the [Segmentation Service](https://experienceleague.adobe.com/en/docs/experience-platform/segmentation/home). <br><br>`segmentNamespaceAllowList` and `segmentNamespaceDenyList` are mutually exclusive."           
```

Comment:
just a super minor update suggestion. 

Restricts users from mapping audiences from the audience namespaces defined in the array to the destination. <br><br> Adobe recommends to allow the export of all audiences, regardless of the origin, by setting `\"segmentNamespaceDenyList\":[]`. [...]